### PR TITLE
Angular update mnrva 430

### PR DESCRIPTION
**Pull request by pras7430**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-430

 ## Description:
Angular version updated to 9
 ## Testing:

 ## Screenshots:
(if applicable)

### CODE REVIEW

File: src/app/app.routing.ts [hunk 1]
```diff
@@ -3,17 +3,17 @@ import { Routes, RouterModule } from '@angular/router';
 import { SchemaResolver } from './_features/monitors/monitor.resolve';
 
 const routes: Routes = [
-  { path: 'resources', loadChildren: './_features/resources/resources.module#ResourcesModule',
+  { path: 'resources', loadChildren: () => import('./_features/resources/resources.module').then(m => m.ResourcesModule),
```
Comment (FarajiA) [inline src/app/app.routing.ts:6]: Oh nice, I wasn't aware the import could be made promise_able_

File: package.json [hunk 2]
```diff
@@ -34,37 +34,37 @@
...
     "chromedriver": "2.45.0",
     "core-js": "2.5.4",
     "hedwig-monitoring-library": "https://github.com/racker/Hedwig/archive/v0.1.0.tar.gz",
     "helix-ui": "0.18.0-rc.2",
     "moment": "2.24.0",
-    "rxjs": "6.3.3",
-    "tslib": "1.9.0",
-    "zone.js": "0.8.26"
+    "rxjs": "6.5.5",
+    "tslib": "^1.10.0",
```
Comment (FarajiA) [inline package.json:54]: Should we remove carat? Unless there's a specific lock we're going for here

File: package.json [hunk 3]
```diff
@@ -34,37 +34,37 @@
...
+    "zone.js": "~0.10.2"
```
Comment (FarajiA) [inline package.json:55]: Same here

File: package.json [hunk 4]
```diff
@@ -34,37 +34,37 @@
...
-    "@angular/language-service": "7.2.0",
+    "@angular-devkit/build-angular": "~0.901.1",
+    "@angular/cli": "9.1.1",
+    "@angular/compiler-cli": "9.1.2",
+    "@angular/language-service": "9.1.2",
     "@compodoc/gulp-compodoc": "0.0.10",
     "@types/jasmine": "2.8.8",
     "@types/jasminewd2": "2.0.3",
-    "@types/node": "8.9.4",
+    "@types/node": "^12.11.1",
```
Comment (FarajiA) [inline package.json:65]: Another

File: package.json [hunk 5]
```diff
@@ -34,37 +34,37 @@
...
+    "rxjs": "6.5.5",
+    "tslib": "^1.10.0",
+    "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.13.9",
-    "@angular/cli": "7.3.9",
-    "@angular/compiler-cli": "7.2.0",
-    "@angular/language-service": "7.2.0",
+    "@angular-devkit/build-angular": "~0.901.1",
```
Comment (FarajiA) [inline package.json:58]: Another

File: package.json (hunk 2 above)
Comment (pras7430) [reply package.json:54]: Yes

File: package.json (hunk 3 above)
Comment (pras7430) [reply package.json:55]: done

File: package.json (hunk 4 above)
Comment (pras7430) [reply package.json:65]: done

File: package.json (hunk 5 above)
Comment (pras7430) [reply package.json:58]: done

File: src/app/app.routing.ts (hunk 1 above)
Comment (pras7430) [reply src/app/app.routing.ts:6]: it is mandatory in angular new version, lazy load module with Dynamic import

**Pull request merged**